node/src/Commands.ts

```diff
@@ -2803,6 +2803,71 @@ export function createMove(
     return createCommand(RequestType.Move, [key, dbIndex.toString()]);
 }
 
+/**
+ * @internal
+ */
+export function createDump(key: GlideString): command_request.Command {
+    return createCommand(RequestType.Dump, [key]);
+}
+
+/**
+ * Optional arguments for `RESTORE` command.
+ *
+ * @See {@link https://valkey.io/commands/restore/|valkey.io} for details.
+ * @remarks `IDELTIME` and `FREQ` modifiers cannot be set at the same time.
```

Comment:
```suggestion
 * @remarks `IDLETIME` and `FREQ` modifiers cannot be set at the same time.
```

Reply:
Same as above:
This isn't enforced anywhere else? Ideally, we prevent the user from setting both at the same time through constructors. Otherwise, we should throw a runtime error if both are defined before sending the request to Valkey.
We should avoid letting Valkey process a command that we know will break.